pkg/util/errutil/status.go

```diff
@@ -88,19 +88,19 @@ func (s CoreStatus) HTTPStatus() int {
 func (s CoreStatus) LogLevel() LogLevel {
 	switch s {
 	case StatusUnauthorized:
-		return LevelInfo
+		return LevelError
 	case StatusForbidden:
-		return LevelInfo
+		return LevelError
 	case StatusNotFound:
-		return LevelDebug
+		return LevelNever
```

Comment:
We already have the router logging so not sure whether we should log these at all?